src/lib/DownloadReachability.js

```diff
@@ -6,7 +6,7 @@ const PROD_CUSTOM_HOST_SUFFIX = 'boxcloud.com';
 const DOWNLOAD_NOTIFICATION_SHOWN_KEY = 'download_host_notification_shown';
 const DOWNLOAD_HOST_FALLBACK_KEY = 'download_host_fallback';
 const NUMBERED_HOST_PREFIX_REGEX = /^https:\/\/dl\d+\./;
```

Comment:
Would we want to update this regex to include `dl-las` as a valid host?

Reply:
Yeah that's a good point, let me update that